CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ project(Umpire
   VERSION 0.3.5)
 
 set(ENABLE_CUDA On CACHE BOOL "")
+set(ENABLE_HIP Off CACHE BOOL "")
```

Comment:
Why is CUDA on by default but not HIP? It would be nice to have it consistent. Both options will probably not be a widely used configuration mode, so maybe default both to off? What are your thoughts @davidbeckingsale and @mcfadden8? Probably best done in a separate PR.

Reply:
@jdahm - they should both be off. Can you create a JIRA ticket?